tests/data_registry/test_views.py

```diff
@@ -11,13 +11,20 @@ class ViewsTests(TestCase):
     def setUpTestData(cls):
         cls.collection = Collection.objects.create(
             title="Dirección Nacional de Contrataciones Públicas (DNCP)",
+            title_en="National Directorate of Public Procurement (DNCP)",
+            title_es="Dirección Nacional de Contrataciones Públicas (DNCP)",
             source_id="paraguay_dncp_records",
             public=True,
         )
         cls.job = cls.collection.job.create(
             active=True,
         )
 
+        Collection.objects.create(
+            title="Non public collection",
+            public=False,
+        )
```

Comment:
Ideally we can test:
* public False, but has active job
* public True, but has no active job
* public False, no active job

We don't need to create these in the setup. We can use [subTest](https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests) to conditionally create them.